io.github.Soundux.yml

```diff
@@ -16,34 +16,50 @@ finish-args:
   - --filesystem=/run/media:ro
   - --filesystem=/media:ro
 modules:
-  - "shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json"
+  - shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json
   - name: libwnck
     buildsystem: meson
     config-opts:
-      - "-Dgtk_doc=false"
+      - -Dgtk_doc=false
     cleanup:
-      - "/include"
-      - "/lib/pkgconfig"
-      - "/bin"
+      - /include
+      - /lib/pkgconfig
+      - /bin
     sources:
-      - type: git
-        url: https://gitlab.gnome.org/GNOME/libwnck.git
-        tag: "40.0"
-        commit: "d161ed31914711b97a1c6ebef0558d80ce51113b"
+      - type: archive
+        url: https://download.gnome.org/sources/libwnck/43/libwnck-43.0.tar.xz
+        sha256: 905bcdb85847d6b8f8861e56b30cd6dc61eae67ecef4cd994a9f925a26a2c1fe
+        x-checker-data:
+          type: gnome
+          name: libwnck
+          versions:
+            '>=': '40.0'
+          url-template: https://download.gnome.org/sources/libwnck/$major/libwnck-$version.tar.xz
+          stable-only: true
   - name: soundux
     buildsystem: cmake-ninja
     config-opts:
       - -DUSE_FLATPAK=ON
       - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_CXX_FLAGS=-Wno-error=deprecated-declarations
     post-install:
-      - "install -Dm 644 -t /app/share/metainfo deployment/appstream/io.github.Soundux.metainfo.xml"
-      - "install -Dm 644 -t /app/share/applications deployment/appstream/io.github.Soundux.desktop"
-      - "install -Dm 644 deployment/flatpak/icons/io.github.Soundux-256.png /app/share/icons/hicolor/256x256/apps/io.github.Soundux.png"
+      - install -Dm 644 -t /app/share/metainfo deployment/appstream/io.github.Soundux.metainfo.xml
+      - install -Dm 644 -t /app/share/applications deployment/appstream/io.github.Soundux.desktop
+      - install -Dm 644 deployment/flatpak/icons/io.github.Soundux-256.png /app/share/icons/hicolor/256x256/apps/io.github.Soundux.png
     sources:
       - type: archive
-        url: https://github.com/Soundux/Soundux/releases/download/0.2.8/soundux-0.2.8.tar.gz
-        sha256: d007841ae5bab71d8a25cc86d27ce137b01b29a7f2b42a5bd62b4ed50dd74043
+        url: https://github.com/Soundux/Soundux/releases/download/0.2.7/soundux-0.2.7.tar.gz
```

Comment:
Do you know if users that currently have 0.2.8 installed even get an update for the flatpak when this is based on 0.2.7 now?

Reply:
flatpak and distro package are two different worlds. so no.

Reply:
Users should get the newest build regardless of the version via Flatpak no?